llarp/service/tag.hpp

```diff
@@ -6,10 +6,7 @@
 #include <util/status.hpp>
 
 #include <sodium/crypto_generichash.h>
-
-#ifndef WIN32_LEAN_AND_MEAN
-#define WIN32_LEAN_AND_MEAN
-#endif
+#include <functional>
```

Comment:
only change to this file, so likely unnecessary new include

Reply:
done